snapshot/windows/windows.go

```diff
@@ -40,45 +79,304 @@ func NewSnapshotter(root string) (snapshot.Snapshotter, error) {
 //
 // Should be used for parent resolution, existence checks and to discern
 // the kind of snapshot.
-func (o *Snapshotter) Stat(ctx context.Context, key string) (snapshot.Info, error) {
-	panic("not implemented")
+func (o *snapshotter) Stat(ctx context.Context, key string) (snapshot.Info, error) {
```

Comment:
nit: can we change `o` to `s`? I have no idea why I had `o` here in the first place, what could it stand for? :thinking:

Reply:
Yeah, `s` makes a lot more sense. Done